micropython/usb/README.md

```diff
@@ -0,0 +1,85 @@
+# Dynamic USB packages
+
+These packages allow implementing USB functionality on a MicroPython device using pure Python code.
```

Comment:
Suggest "MicroPython device" -> "MicroPython system" to avoid confusion with device meaning USB device.